### DISCUSSION TITLE
When developing front-end locally, how can you access the back-end services? Can you access them through Docker, or do you need to deploy the server-side?

### DISCUSSION THREAD

**Discussion by xiaomingcimy** [langgenius/dify#6700] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

When developing front-end locally, how can you access the back-end services? Can you access them through Docker, or do you need to deploy the server-side?

**Comment by crazywoola** [comment]
Both, you can startup the server in source code or using the docker image, you can setup the `,env` file under `web/`
